### PR TITLE
Add "make me-a-devenv-pls"

### PR DESCRIPTION
**Pull request by odinuge**
Getting tired of fighting with python deps when developing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,7 @@ help:
 	@echo 'docs                   - build and display docs'
 	@echo 'ci_settings            - create a lego/settings/local.py for ci'
 	@echo 'fixme                  - Fix code formatting'
+	@echo 'me-a-devenv-pls        - yes. It does that.'
```
Comment (SmithPeder) [inline Makefile:6]: rename to devenv

**Review by SmithPeder — COMMENTED** [review]
Nice, but rename to something more intuitive

File: Makefile (hunk 1 above)
Comment (odinuge) [reply Makefile:6]: Just feels like it comes with way more guarantees if the name isn't silly

Comment (odinuge) [reply Makefile:6]: But I guess I can rename

**Comment by odinuge** [comment]
Did you try this on mac @SmithPeder? Not sure if `--net=host` works properly there..

**Comment by SmithPeder** [comment]
ofc not, can try tho 🙃

**Comment by odinuge** [comment]
ping @SmithPeder

**Comment by SmithPeder** [comment]
I gotta break it to you, #no_mac

**Pull request merged**